src/main/java/com/ericsson/ei/jsonmerge/MergeHandler.java

```diff
@@ -154,23 +155,20 @@ private void updateJsonObject(JSONObject aggregatedJsonObject, JSONObject prepar
             while (preparedJsonKeys.hasNext()) {
                 String preparedJsonKey = preparedJsonKeys.next();
                 if (aggregatedJsonObject.has(preparedJsonKey)) {
-                    Class valueClass = aggregatedJsonObject.get(preparedJsonKey).getClass();
-                    if (valueClass.equals(JSONObject.class)) {
-                        updateJsonObject((JSONObject) aggregatedJsonObject.get(preparedJsonKey),
-                                (JSONObject) preparedJsonObject.get(preparedJsonKey));
-                        // final JSONObject o1 = getAsT(aggregatedJsonObject,
-                        // preparedJsonKey, JSONObject.class);
-                        // final JSONObject o2 = getAsT(preparedJsonObject,
-                        // preparedJsonKey, JSONObject.class);
-                        // updateJsonObject(o1, o2);
-                    } else if (valueClass.equals(JSONArray.class)) {
-                        updateJsonObject((JSONArray) aggregatedJsonObject.get(preparedJsonKey),
-                                (JSONArray) preparedJsonObject.get(preparedJsonKey));
-                        // final JSONArray a1 = getAsT(aggregatedJsonObject,
-                        // preparedJsonKey, JSONArray.class);
-                        // final JSONArray a2 = getAsT(preparedJsonObject,
-                        // preparedJsonKey, JSONArray.class);
-                        // updateJsonObject(a1, a2);
+                    final Object eFromAgg = aggregatedJsonObject.get(preparedJsonKey);
```

Comment:
What does this variable contain? I don't think eFromAgg is very descriptive.

Reply:
Done